gp2gp_adaptor_response_docs.yaml

```diff
@@ -0,0 +1,557 @@
+openapi: 3.0.1
+info:
+    title: OpenAPI definition
+    version: v0
+servers:
+    - url: http://localhost:8080/api
+      description: Generated server url
+paths:
+    /requests:
+        post:
+            tags:
+                - Ehr Status requests controller
+            operationId: getEhrRequestsEncodedForm
+            parameters:
+                - name: request
+                  in: query
+                  required: true
+                  schema:
+                      $ref: '#/components/schemas/EhrStatusRequestQuery'
+            requestBody:
+                content:
+                    application/json:
+                        schema:
+                            $ref: '#/components/schemas/EhrStatusRequestQuery'
+                        examples:
+                            request-example:
+                                summary: Example Request for EHR Status
+                                value:
+                                    fromDateTime: "2020-10-31T01:30:00.000Z"
+                                    toDateTime: "2030-10-31T01:30:00.000Z"
+                                    fromAsid: "918999198738"
+                                    toAsid: "200000000359"
+                                    fromOdsCode: "P883302"
+                                    toOdsCode: "P113302"
+                required: true
```

Comment:
Hmm, I'm pretty sure I've also called this endpoint without specifying a request body and it's worked. 🤔